app/_locales/en/messages.json

```diff
@@ -1297,8 +1297,8 @@
     "message": "verify the network details",
     "description": "Serves as link text for the 'mismatchedChain' key. This text will be embedded inside the translation for that key."
   },
-  "mobileSyncText": {
-    "message": "Please enter your password to confirm it's you!"
+  "mobileSyncWarning": {
+    "message": "⚠️ Proceeding will show a code that allows exporting all of your accounts. Support staff will never request this. DO NOT SHARE!"
```

Comment:
I don't want to quibble over this urgent fix, but on the other hand my bank doesn't yell at me in all caps and neither should we:

```suggestion
    "message": "⚠️ Proceeding will display a secret QR code that allows access to your accounts. Do not share it with anyone. Support staff will never ask your for it."
```

Reply:
I agree. I think I took the all caps from another suggestion, even though I agree it rubs against our established text language (have we written that anywhere? I feel like Rachel helped set that standard)